lib/spectrum/response/specialists.rb

```diff
@@ -92,7 +92,8 @@ def fetch_records(query)
           fq: query[:fq],
           qq: '"' + RSolr.solr_escape(query[:q]) + '"',
           rows: rows.first,
-          fl: fields.first
+          fl: fields.first,
+          df: query[:df] || 'allfields'
```

Comment:
Do we know what `query` is here (type of object) -- is it just a hash?

Reply:
Yep, just a hash. It gets created on line 235ish in this file.